xbmc/powermanagement/PowerManager.cpp

```diff
@@ -298,6 +301,8 @@ void CPowerManager::OnWake()
   g_weatherManager.Refresh();
 
   CServiceBroker::GetPVRManager().OnWake();
+  g_application.ResumePlayer(m_lastItemFile);
```

Comment:
wouldn't it make sense to first check if you have a valid m_lastItemFile before you trigger ResumePlayer()? I mean, why trigger the process if nothing was playing before standby etc

Reply:
@da-anda please don't confuse people here. you said yourself that you are not familiar with the code. please refrain from commenting here.

Reply:
sorry, my bad

Reply:
I guess it's checked somewhere very deep in the player prepeation.
I will add a new item is empty check to detect resettet items and use it inside of ResumePlayer